build/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.13.6 as builder
+WORKDIR /app
+COPY ./ ./
+ARG NAME
+ARG VERSION
+RUN GOOS=linux GOARCH=amd64 go build -o canary-checker -ldflags "-X \"main.version=$VERSION\""  main.go
+
+FROM golang:1.13.6
+LABEL maintainer="Vikas Saini"
```

Comment:
* Remove maintainer label
* Move Dockerfle to root directory

Reply:
@moshloop Done